server/src/main/java/org/elasticsearch/node/NodeConstruction.java

```diff
@@ -538,6 +540,17 @@ private SettingsModule validateSettings(Settings envSettings, Settings settings,
         return settingsModule;
     }
 
+    @UpdateForV9
+    private static void addBwcSearchWorkerSettings(List<Setting<?>> additionalSettings) {
+        // TODO remove the below settings, they are unused and only here to enable BwC for deployments that still use them
```

Comment:
this may deserve some better annotation than a TODO, perhaps `@UpdateForV9`

Reply:
it already has @UpdateForV9, but maybe you added that.

Reply:
oh boy, I have already done that then... but I did not see it when I reviewed. all good